tools/bundle_adjust_tracks.cxx

```diff
@@ -180,9 +180,18 @@ static kwiver::vital::config_block_sptr default_config()
                     "when updating cameras. This option is only relevent if a "
                     "value is give to the input_pos_files option.");
 
-
       config->set_value("krtd_clean_up", "false",
                         "Delete all previously existing KRTD files present in output_krtd_dir before writing new KRTD files.");
+
+  config->set_value("depthmaps_points_file", "",
```

Comment:
> Maybe the better approach is to create a template for a GUI config files that contains only the config options that are loaded by the GUI along with comments describing what the options do. We could put this file in the config or maybe examples directory so it would be installed with MAP-Tk. Then users could use this template to manually provide what is needed to load all the content for their project.

I agree, it's a better idea. It could be also used for the volume & mesh integration in the future, or a config file for it could be created as well.